cluster/openvsx/extension-uploader/importvsx.sh

```
@@ -74,22 +72,28 @@ function getCompatibleRelease() {
   local index=0
+  local offset=0
+  local num_releases=0
 
-  local ext_data=$(curl -X GET "https://open-vsx.org/api/v2/-/query?extensionId=${ext_id}&includeAllVersions=true&targetPlatform=${ARCH}&size=${QUERY_RESULT_SIZE}&offset=0" -H 'accept: application/json')
-  results=$(echo "${ext_data}" | jq -r '.totalSize')
-  if [[ ${results} -eq 0 ]]
-  then
-    ext_data=$(curl -X GET "https://open-vsx.org/api/v2/-/query?extensionId=${ext_id}&includeAllVersions=true&targetPlatform=universal&size=${QUERY_RESULT_SIZE}&offset=0" -H 'accept: application/json')
+  while [[ ${num_releases} == 0 ]]
+  do
+    local ext_data=$(curl -X GET "https://open-vsx.org/api/v2/-/query?extensionId=${ext_id}&includeAllVersions=true&targetPlatform=${ARCH}&size=${QUERY_RESULT_SIZE}&offset=${offset}" -H 'accept: application/json')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)